### ISSUE TITLE
"Briefly describe the issue" does not accept any value [New UI]

### ISSUE THREAD

**Issue by mcroud** [webcompat/webcompat.com#2975]
On Step 2 (Issue), I select "Something else" from the options.
I enter some text in the "Briefly describe the issue:" text field  which appears.

No matter what I enter, the text field is not validated and the "Confirm" button remains disabled.

<img width="1090" alt="Screenshot 2019-10-11 at 16 49 27" src="https://user-images.githubusercontent.com/42435754/66665798-2d65f300-ec47-11e9-8eee-a271039145e4.png">

**Comment by bacharakis** [comment]
Assigned to @munteanu210

**Comment by munteanu210** [comment]
I fixed this in PR #2991. 

I added validation for the field. When this field is empty, the "Continue" button is disabled. ![continue-is-disabled](https://user-images.githubusercontent.com/1682059/67230685-917c7a00-f3f2-11e9-827c-b4ee4b929533.png)

When you enter something, the field is validated, the check symbol appears and the button is activated to proceed to the next step. ![confirm-enabled](https://user-images.githubusercontent.com/1682059/67230763-c38ddc00-f3f2-11e9-90c3-8cfae67589f9.png)
